internal/generate/generate_modules.go

```diff
@@ -65,9 +74,17 @@ type TemplateConfig struct {
 	isTemplate  bool
 }
 
+type BinFileConfig struct {
+	source      string
+	destination string
+	mineType    string
```

Comment:
Typo

Reply:
Any reason to have the mime type here? Or just because it was available?

Reply:
just because it's available; if we'd like to post-process on filter on it in the future.

Reply:
Actually "isTemplate" in TemplateConfig is not even used anymore. How about we just have a struct with source and destination and use that for both return lists?